src/Sylius/Component/Core/Model/OrderItemUnit.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Model;
+
+use Sylius\Component\Inventory\Model\InventoryUnitInterface;
+use Sylius\Component\Inventory\Model\StockableInterface;
+use Sylius\Component\Order\Model\OrderItemUnit as BaseOrderItemUnit;
+use Sylius\Component\Shipping\Model\ShipmentInterface as BaseShipmentInterface;
+use Sylius\Component\Shipping\Model\ShippableInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class OrderItemUnit extends BaseOrderItemUnit implements OrderItemUnitInterface
+{
+    /**
+     * @var string InventoryUnitInterface::STATE_*
+     */
+    protected $inventoryState = InventoryUnitInterface::STATE_CHECKOUT;
+
+    /**
+     * @var BaseShipmentInterface
+     */
+    protected $shipment;
+
+    /**
+     * @var string BaseShipmentInterface::STATE_*
+     */
+    protected $shippingState = BaseShipmentInterface::STATE_CHECKOUT;
+
+    /**
+     * @var \DateTime
+     */
+    protected $createdAt;
+
+    /**
+     * @var \DateTime
+     */
+    protected $updatedAt;
```

Comment:
I've discussed it with @pjedrzejewski and we think that for now there is no need to make ``OrderItemUnit`` timestampable in *Order* component, however, it have to has these fields in *Core*, as ``OrderItemUnitInterface`` extends ``InventoryUnitInterface``.